docs/conf/helpop-full.conf.example

```diff
@@ -927,11 +930,39 @@ using their cloak when they quit.">
                     channel (requires stripcolor module).
  T                  Blocks /NOTICEs to the channel from users who are
                     not at least halfop (requires nonotice module).
-
- g <mask>           Blocks messages matching the given glob mask
-                    (requires chanfilter module).
- X <mode>           Makes channel operators immune to the specified
-                    restrictive mode (requires exemptchanops module).
+ X <mode>:<status>  Makes users of <status> or higher immune to the
+                    specified restrictive mode. For example: flood:h
+                    (requires exemptchanops module).
+Possible restrictive modes for +X are:
```

Comment:
This should have a space before it and should probably read

>Valid exemptions ???s for +X are:

(See the comment above for what to put instead of ???.)